apis/v1alpha3/backendtlspolicy_types.go

```diff
@@ -73,16 +74,26 @@ type BackendTLSPolicySpec struct {
 	//   be unique across all targetRef entries in the BackendTLSPolicy.
 	// * They select different sectionNames in the same target.
 	//
+	// This may be left unset for reference from a Route filter.
+	//
 	// Support: Extended for Kubernetes Service
 	//
 	// Support: Implementation-specific for any other resource
 	//
-	// +kubebuilder:validation:MinItems=1
+	// +optional
+	// +kubebuilder:validation:MinItems=0
```

Comment:
This would make the BackendTLSPolicy no longer a "policy", according to definition.

Reply:
+1.  Again, please propose the changes in the GEP and not directly in the API.